jsk_2016_01_baxter_apc/test/test-ik-in-bin.l

```diff
@@ -35,18 +44,9 @@
             (setq pos (float-vector tmp-x tmp-y tmp-z))
             (assert
               (or
-                (send robot :rarm :inverse-kinematics
-                  (make-cascoords :pos pos)
-                  :rotation-axis t
-                  :debug-view nil
-                  :warnp nil
-                  :dump-command nil)
-                (send robot :larm :inverse-kinematics
-                  (make-cascoords :pos pos)
-                  :rotation-axis t
-                  :debug-view nil
-                  :warnp nil
-                  :dump-command nil)
+                (ik-with-rotating-gripper robot :rarm 90 (make-cascoords :pos pos))
+                (ik-with-rotating-gripper robot :rarm 0 (make-cascoords :pos pos :rpy (float-vector 0 (deg2rad -90) 0)))
+                (ik-with-rotating-gripper robot :larm 90 (make-cascoords :pos pos))
```

Comment:
Sorry for confusing. For me, gripper-angle is not the angle of rotating gripper. It is the angle between z-axis of gripper coordinate and z-axis of hand coordinate(e.g. `right_hand` link).
In right gripper, that indicates the rotating angle from unbended pose. In left gripper, that is fixed to 90 degrees.